zulipterminal/model.py

```diff
@@ -438,6 +425,22 @@ def fetch_custom_emojis(self) -> NamedEmojiData:
         display_error_if_present(response, self.controller)
         return custom_emojis
 
+    def generate_all_emoji_data(self) -> NamedEmojiData:
+        unicode_emoji_data = unicode_emojis.EMOJI_DATA
+        for name, data in unicode_emoji_data.items():
+            data['type'] = 'unicode_emoji'
+        typed_unicode_emoji_data = cast(NamedEmojiData, unicode_emoji_data)
+        custom_emoji_data: NamedEmojiData = self.fetch_custom_emojis()
+        zulip_extra_emoji: NamedEmojiData = {
+                'zulip': {'code': 'zulip', 'type': 'zulip_extra_emoji'}
+        }
+        all_emoji_data = {**typed_unicode_emoji_data,
+                          **custom_emoji_data,
+                          **zulip_extra_emoji}
```

Comment:
If you leave the `.items()` here then the 'move diff' will be smaller.

In case you didn't see my note about noting colored blocks try looking at the diff for just this commit using this:
https://git-scm.com/docs/diff-options#Documentation/diff-options.txt---color-movedltmodegt
(or the equivalent in a .gitconfig or similar)